doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,17 +169,78 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Load Balancer platform service.
+
++------------+--------------------------------------------------------+---------------------------------------------+
+| Reference  | Requirement                                            | Notes                                       |
++============+========================================================+=============================================+
+| pas.lb.001 | The Load Balancer must support resource scaling        |                                             |
++------------+--------------------------------------------------------+---------------------------------------------+
+| pas.lb.002 | The Load Balancer must support resource resiliency     |                                             |
++------------+--------------------------------------------------------+---------------------------------------------+
+| pas.lb.003 | The Load Balancer must support scaling and resiliency  | Define local environment: within a subnet,  |
+|            | in the local environment                               | tenant network, Availability Zone of a      |
+|            |                                                        | cloud, ...                                  |
```

Comment:
"Define local environment" reads like a requirement on the load balancer ;-).

Reply:
@ulikleber It is